dist2src/worker/sentry.py

```diff
@@ -4,19 +4,16 @@
 from logging import getLogger
 from os import getenv
 
-from dist2src.worker.decorators import only_once
+from dist2src.worker.decorators import only_once, if_sentry_is_configured
 
 logger = getLogger(__name__)
 
 
 @only_once
+@if_sentry_is_configured
 def configure_sentry(runner_type: str) -> None:
```

Comment:
I'd just name the decorator class differently because I read this as "configure sentry if sentry is configured".
`@if_sentry_env_vars_set` perhaps?

Reply:
Good point, I missed that. How about `if_sentry_is_enabled`? Then it would read "configure sentry if sentry is enabled".

The reason I would go with this is that we might choose not to enable sentry by setting some env vars in the future.

WDYT?

Reply:
sound OK to me